### DISCUSSION TITLE
internet usage

### DISCUSSION THREAD

**Discussion by SalimiHabib** [Chia-Network/chia-blockchain#6986] [Support]
After reviewing the cost of the Internet this month, I realized that a very large amount of this cost is related to chia application, and a closer look revealed that the amount of consumption is one hundred megabytes per hour or  two and a half gigabytes per day.
Is this number correct ?!
Is there a way to reduce this amount of internet usage?
In fact, chia`s  income is very low compared to its internet cost

**Comment by loppefaaret** [comment]
perhaps lessen the amount of connections you got to the network, is the solution you are looking for ? 
these 2 lines in `config.yaml` located under `C:\Users\<USERNAME>\.chia\mainnet\config\` is controlling that:
```
  target_outbound_peer_count: 10
  target_peer_count: 80
```
first line is how many your peer actively try to maintain
second line is the absolute max it accepts in total (incoming)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
